src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -36,48 +34,41 @@ export class TableInteractorColumnComponent implements OnInit {
     }
   }
 
-  public stochiometryOfInteractors(complex, componentId): string {
-    const match = complex.components.find(component => component.id === componentId);
+  public stochiometryOfInteractors(complex: Element, componentId: string): string {
```

Comment:
I have added types for every method parameter to make sure all the types and fields used are correct and the IDE checks that.